gen-apidocs/generators/html.go

```diff
@@ -571,29 +565,35 @@ func (h *HTMLWriter) generateHTML(navContent string) {
 
 	for _, sec := range h.TOC.Sections {
 		fmt.Printf("Collecting %s ... ", sec.File)
-		content, err := ioutil.ReadFile(filepath.Join(*api.ConfigDir, "includes", sec.File))
+		content, err := ioutil.ReadFile(filepath.Join(api.IncludesDir, sec.File))
```

Comment:
@tengqm , Check where the `config/sections` directory variable is set?

Reply:
This has nothing to do with section files. The so called "sections" are actually the old "static_includes".

Reply:
Right. I am trying to track down where the files are read/copied from the new dirs `config/sections` as well as `config/examples`.

Reply:
Never mind. Found it.
The changes seem fine now. Have you rebuilt the API reference?

Reply:
No. Haven't done that.